### DISCUSSION TITLE
Where should to put driftRuntimeOptions ?

### DISCUSSION THREAD

**Discussion by CharlotteJu** [simolus3/drift#2924] [Q&A] [answered]
Hi, 
I have warnings in my code because I restart my database instance, and I saw that I had to set "driftRuntimeOptions.dontWarnAboutMultipleDatabases = true" to remove it. Except I don't understand where to put it, can you help me :) ?

**Comment by simolus3** [comment, marked as answer]
If you know it's safe, you could expose your database class as a `factory` and just set it there before creating the database instance. Enabling it in `main` or any other method you call before using the database would also work.
